### ISSUE TITLE
Install Lagoon in all Lagoon CLI Containers

### ISSUE THREAD

**Issue by Schnitzel** [amazeeio/lagoon-cli#45]
**Is your feature request related to a problem? Please describe.**
Feature Request

**Describe the solution you'd like**
Have `lagoon` cli already installed in all CLI Images of Lagoon

**Comment by GROwen** [comment]
This would be great for a whole bunch of reasons. The primary one I'm after is being able to trigger a `Drush_sql-dump` task during the pre-deploy step so that the backup is available via the UI.

Our project currently has a solution to dump this to the filesystem but we have vendors who would appreciate access via the UI.

**Comment by vincenzodnp** [comment]
What is the best option in your opinion:
- all cli images
- just toolbox image
@twardnw @shreddedbacon @tobybellwood @Schnitzel

**Comment by Schnitzel** [comment]
@vincenzodnp I wrote my opinion in the first comment already, plus also in the header of this issue.
>  Have lagoon cli already installed in all CLI Images of Lagoon

Not sure why we would install it only in the toolbox? as the toolbox is not used by customers it wouldn't really help them?

**Comment by twardnw** [comment]
Yes, installed in CLI images so that it is easy for customers to use

**Comment by bomoko** [comment]
@shreddedbacon - so I started playing around with this inside the containers and it looks like, by _default_ the dynamically linked binaries don't work straight (at least in alpine).

Building statically, it works just peachy. I wonder if it doesn't make sense to do a static release to support people running the cli inside containers?

**Comment by shreddedbacon** [comment]
> @shreddedbacon - so I started playing around with this inside the containers and it looks like, by _default_ the dynamically linked binaries don't work straight (at least in alpine).
> 
> Building statically, it works just peachy. I wonder if it doesn't make sense to do a static release to support people running the cli inside containers?

@bomoko definitely. If you want to submit a PR to update the Makefile to support static builds, I'm down for it.
